libs/ui/src/button.tsx

```diff
@@ -43,8 +43,8 @@ const buttonVariants = {
 export type ButtonVariant = keyof typeof buttonVariants;
 export const BUTTON_VARIANTS = Object.keys(buttonVariants) as ButtonVariant[];
 
-type ClickHandler = () => void;
-type TypedClickHandler<T> = (value: T) => void;
+export type ClickHandler = (event: SyntheticEvent) => void;
+export type TypedClickHandler<T> = (value: T, event: SyntheticEvent) => void;
```

Comment:
Maybe a more specific event type would be appropriate here. E.g.:

```ts
type ButtonPressEvent = React.TouchEvent<ButtonElement> | React.MouseEvent<ButtonElement>
```

That way the consumer knows what fields it can access more precisely.

Reply:
Good suggestion, thanks! [635c445](https://github.com/votingworks/vxsuite/pull/4284/commits/635c44510fb64fa932b7781a8880fd7c76667521)